### PR TITLE
NN HW07

### PR DESCRIPTION
**Pull request by Sol-Arina**

### CODE REVIEW

**Review by fortvivlan — APPROVED** [review]
```
loss_fn = nn.CrossEntropyLoss()
loss = loss_fn(logits, labels)
```
->
`loss = nn.functional.cross_entropy(logits, labels)`

---

Сложный какой-то инференс, датасет из одного примера и вот это все... почему бы просто не:
```
tokenizer = BertTokenizer.from_pretrained('bert-base-cased')
text = "Replace me by any text you'd like."
encoded_input = tokenizer(text, return_tensors='pt')
output = model(**encoded_input)
```
?
